### PR TITLE
Optout: UserId module and Sharedid submodule

### PR DESCRIPTION
**Pull request by SKOCHERI**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by SKOCHERI** [comment]
The shared Id changes are in pull request https://github.com/prebid/shared-id/pull/10

File: modules/sharedIdSystem.js [hunk 1]
```diff
@@ -10,9 +10,13 @@ import {ajax} from '../src/ajax.js';
 import {submodule} from '../src/hook.js';
 
 const MODULE_NAME = 'sharedId';
-const ID_SVC = 'https://id.sharedid.org/id';
+const SHAREDID_URL = 'https://id.sharedid.org';
+const ID_SVC = SHAREDID_URL + '/id';
+const SYNC_SVC = SHAREDID_URL + '/iframe/optout';
```
Comment (smenzer) [inline modules/sharedIdSystem.js:15]: I get a 404 when calling https://id.sharedid.org/iframe/optout ... is this ready to go?

**Review by smenzer — CHANGES_REQUESTED** [review]
Can you please explain the workflow here? You didn't provide a PR description and I want to make sure your intentions align to how the code is written and what I understand from it.

I also don't see any tests related to the actual optout method in user id module, so can you please add coverage?

Thanks!

File: modules/sharedIdSystem.js (hunk 1 above)
Comment (smenzer) [reply modules/sharedIdSystem.js:15]: never mind...I missed this comment, sorry! https://github.com/prebid/Prebid.js/pull/6068#issuecomment-736902048

**Comment by ChrisHuie** [comment]
@SKOCHERI is this pr still relevant or active or can I close it out?

**Comment by SKOCHERI** [comment]
> @SKOCHERI is this pr still relevant or active or can I close it out?

Closing this pull request

**Pull request closed without merging**